src/lib/components/Sidebar.svelte

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+  import QRCodeIcon from "$lib/images/qr-code-icon.svg"
+  import SchemasIcon from "$lib/images/schemas-icon.svg"
+  import HomeIcon from "$lib/images/home-icon.svg"
+  import { page } from "$app/stores"
+
+  export let mobile = false
+</script>
+
+<nav class="sidebar" class:mobile-only={mobile}>
+  <a href="/" class="menu-item" class:selected={$page.url.pathname === "/"}>
+    <div class="icon-wrapper">
+      <HomeIcon />
+    </div>
+    <div class="name">Home</div>
+  </a>
+  <a href="/tags" class="menu-item" class:selected={$page.url.pathname.startsWith("/tags")}>
+    <div class="icon-wrapper">
+      <QRCodeIcon />
+    </div>
+    <div class="name">IOXIO Tags™</div>
+  </a>
+  <a href="/schemas" class="menu-item" class:selected={$page.url.pathname.startsWith("/schemas")}>
+    <div class="icon-wrapper">
+      <SchemasIcon />
+    </div>
+    <div class="name">Schemas</div>
+  </a>
+</nav>
+
+<style lang="scss">
+  @import "$styles/setup.scss";
+
+  nav {
+    width: 72px;
```

Comment:
This area is now mixing both `px` and `rem` for layout, `.menu-item` `margin` uses `rem`, `.icon-wrapper` `padding` uses `px`and this width uses `px` or `rem` depending on which state it is in, this seems like a recipe for problems.